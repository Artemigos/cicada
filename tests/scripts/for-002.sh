echo hi
for xxx in foo bar baz
    echo $xxx
done

for xxx in $(echo a b)
    echo hello && echo $xxx
done

for xxx in 'args kwargs' "sh script"
    echo $xxx
done

echo bye
