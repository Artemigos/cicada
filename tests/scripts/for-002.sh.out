hi
foo
bar
baz
hello
a
hello
b
args kwargs
sh script
bye
